src/odoc/etc/odoc.css

```diff
@@ -392,7 +392,7 @@ pre code {
 
 /* Module member specification */
 
-.spec:not(.include), .spec.include details summary {
+.spec {
```

Comment:
For people devising markup this is a good example of why classes should not be used for vastly different contexts.

Previous to this commit the `spec include` classes were used on a div wrapping the whole include and its markup content. But otherwise the `spec` class was always used only on `div`s simply wrapping the source of structure items. This entailed this incredibly complex selector for styling structure items sources which, with the new structure, now boils down to a simle `.spec`.